runtime/src/governance/proposals.rs

```diff
@@ -143,6 +143,9 @@ decl_event!(
 
         /// * Hash - hash of wasm code of runtime update.
         RuntimeUpdated(u32, Hash),
+
+        /// Root cancelled proposal
+        ProposalVeto(u32),
```

Comment:
It would be better to call it `ProposalVetoed` so that event name describes _what happened_